azext_iot/tests/iothub/test_iot_ext_int.py

```diff
@@ -32,10 +33,28 @@
 
 CWD = os.path.dirname(os.path.abspath(__file__))
 
+test_managed_name = "TESTMANAGEDIDENTITY"
```

Comment:
Would be good to slightly tweak the current `LIVE_STORAGE` and `LIVE_STORAGE_ID` variables to be more explicit/reduce reliance on comment. Maybe `LIVE_STORAGE` -> `LIVE_STORAGE_URI` || `LIVE_STORAGE_SAS_URI`  and `LIVE_STORAGE_ID` -> `LIVE_STORAGE_RESOURCE_ID`.